php-classes/Emergence/People/InvitationsRequestHandler.php

```diff
@@ -29,6 +29,13 @@ public static function handleRequest()
                 return static::handleSendRequest();
             case 'accept':
                 return static::handleAcceptRequest();
+            case '*userclasses':
```

Comment:
@nbey PeopleRequestHandler already offers and endpoint that might be useful for this. It doesn't filter out < User for you, but maybe that could be a query string option to the existing endpoint?

I just added an IUser interface and added a `?interface=user` filter for `/people/*classes`: http://v2.slate.is/people/*classes?format=json&interface=user

Also, multi-word routes should use hyphens